app/services/issue_card_cloner.rb

```diff
@@ -9,7 +9,9 @@ def initialize(repo, template_card, cloned_column, client)
 
   def run
     create_issue!
+    sleep(1)
     create_card!
+    sleep(1)
```

Comment:
this is okay for now, but ultimately I'd love to look into exponential backoff instead of hard coding sleeps... instead we should be catching rate limit exceptions and sleeping then for increasing periods of time until things are cooled down.

Reply:
The Github docs say that we should wait at least one second after creating something, so I think we should always do the 1 second wait in addition to catching exceptions and triggering exponential backoff if the 1 second delay is not enough. What do you think of that?

Reply:
I think it's fine for now